File: Client/src/app/book/book.ts
export interface Book {
  id: number;
  name: string;
  totalQuantity: number;
  availableQuantity: number;
  description: string;
  author: string;
  coverImage: string;
  publisher: string;
  isbn: number;
  publicationDate: Date;
  pages: string;
  maxRows: number;
}

File: API/Models/Book.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models
{
    public class Book
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public int TotalQuantity { get; set; }
        public int AvailableQuantity { get; set; }
        public string? Author {get;set;}
        public string? Description {get;set;}
        public string? CoverImage {get;set;} 
        public string? Publisher {get;set;}
        public int ISBN {get;set;}
        public DateTime? PublicationDate {get;set;}
        public int Pages {get;set;}
        [NotMapped]
        public int MaxRows { get; set; }
        
    }
}
